doc/Tex/RAM_SCB_run.tex

```diff
@@ -1,95 +1,48 @@
 %\section{Run Directories \label{subchap:rundir}}
-Running RAM-SCB occurs not in the installation directory, but through 
-\textit{run directories} -- special directories that keep your simulations 
-separate from the installation and other simulations.  To create a run 
-directory, simply use the {\tt make} interface:
-
+Running RAM-SCB occurs not in the installation directory, but through \textit{run directories} -- special directories that keep your simulations separate from the installation and other simulations. To create a run directory, simply use the {\tt make} interface:
 \begin{verbatim}
 make rundir
 \end{verbatim}
 \noindent
-{\tt make} will unpack some default inputs and organize a new directory
-aptly named {\tt run}.  Note that if a run directory exists that is called
-``run'', RAM-SCB will not over write it and {\tt make} will complain.  
-
-The first thing you should do is rename your run directory and move it to
-a place that makes sense given the constraints of your system.  For example,
-let's pretend we're using NASA's Pleiades system and doing a set of validation
-simulations.  On this system, you typically install code on your home 
-directory, but must run from designated directories called ``nobackupXX'' 
-(where XX is the number of the directory and nobackup tells you why you 
-didn't install your code there.)  From the install directory, make and move
-your first run directory:
+{\tt make} will unpack some default inputs and organize a new directory aptly named {\tt run}. Note that if a run directory exists that is called ``run'', RAM-SCB will not over write it and {\tt make} will complain. 
 
+The first thing you should do is rename your run directory and move it to a place that makes sense given the constraints of your system. For example, let's pretend we're using NASA's Pleiades system and doing a set of validation simulations. On this system, you typically install code on your home directory, but must run from designated directories called ``nobackupXX'' (where XX is the number of the directory and nobackup tells you why you didn't install your code there.) From the install directory, make and move your first run directory:

```

Comment:
Again with the Pleiades... This is a generic "on HPC you should run on scratch" statement, that we should probably make generically.

Reply:
K